draft-ietf-teep-architecture.md

```diff
@@ -166,6 +166,11 @@ The Trusted Execution Provisioning (TEEP) protocol addresses the following probl
     between cooperating TAs under the SP's control, and specify whether
     the TAs can communicate, share data, and/or share key material.
 
+Note: The Service Provider requires the help of a TAM in order to provision
```

Comment:
Would be better grammar to drop "in order"

Reply:
Deleted the term 'in order'.